src/components/calculator/Calculator.css

```diff
@@ -0,0 +1,61 @@
+button {
+  padding: 1.5rem 2.5rem;
+  font-size: 1.3rem;
+  font-weight: bold;
+  background-color: #fff;
+  border: 1px solid rgb(241, 238, 238);
+  cursor: pointer;
+}
+
+button:hover {
+  background-color: rgba(255, 165, 0, 0.5);
+}
+
+.calc-body {
+  display: flex;
+  flex-flow: column;
+  list-style-type: none;
+  border: 2px solid rgb(241, 238, 238);
+  width: 40%;
+  padding: 0;
+  margin: 150px auto 0;
+}
+
+.calc-screen {
+  text-align: end;
+  padding: 30px;
+  background-color: rgb(241, 238, 238);
+  font-weight: bold;
+  font-size: 1.5rem;
+}
```

Comment:
- [ ] [OPTIONAL] Great project you have here @levy002 , however, for a better UI, you need to ensure the screen aligns properly with the body of the calculator (screen width) just as it is in the [bigger picture](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/sneak_peek.md)
### Yoru Implementation
![Screenshot from 2022-04-18 23-23-44](https://user-images.githubusercontent.com/58692590/163887810-612ee561-8d9f-4c6a-8866-83e5a3d976c3.png)

Reply:
Thanks, @Whoistolu for suggesting this change. It's solved now!